README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio
+# Microverse
 
 > Create Professional Portifolio
```

Comment:
[ OPTIONAL ] README should have a description of what this project is about and its features, and instructions on how to set the project up locally.